README.md

```diff
@@ -117,20 +117,20 @@ MarkerManager markerManager = // ...
 MarkerManager.Collection collection = markerManager.newCollection();
 
 // Set InfoWindowAdapter and OnInfoWindowClickListener
-collection.setInfoWindowAdapter(apdapter);
+collection.setInfoWindowAdapter(adapter);
 collection.setOnInfoWindowClickListener(listener);
 
 // Alternatively, if you are using clustering
 ClusterManager<ClusterItem> clusterManager = // ...
-ManagerManager.Collection markerCollection = markerCollection.setInfoWindowAdapter(apdapter);
+MarkerManager.Collection markerCollection = markerCollection.setInfoWindowAdapter(adapter);
 markerCollection.setOnInfoWindowClickListener(listener);
 ```
 
 _Old_
 ```java
 CustomInfoWindowAdapter adapter = // ...
 OnInfoWindowClickListener listener = // ...
-googleMap.setInfoWindowAdapter(apdapter);
+googleMap.setOnInfoWindowAdapter(adapter);
```

Comment:
See https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap#setInfoWindowAdapter(com.google.android.gms.maps.GoogleMap.InfoWindowAdapter), this should be:
`googleMap.setInfoWindowAdapter(adapter);`

```suggestion
googleMap.setInfoWindowAdapter(adapter);
```